uber/config.py

```diff
@@ -165,8 +174,33 @@ def SUPPORTERS_ENABLED(self):
 
     @property
     def AT_THE_DOOR_BADGE_OPTS(self):
-        opts = [(self.ATTENDEE_BADGE, 'Full Weekend Pass (${})'.format(self.BADGE_PRICE))]
-        if self.ONE_DAYS_ENABLED:
+        """
+        This populates the dropdown form on the /registration/register page with a list of badges available at-door.
+        It always offers a "Full Weekend Pass" if attendee badges are available.
+        If one-days are enabled, it will show either a generic "Single Day Pass" or a list of specific day badges,
+        depending on if PRESELL_ONE_DAYS is true. Each day's badge will need to be configured under [badge_ranges] and
+        [[badge]] to show up in this dropdown.
+        """
+        opts = []
+        if self.ATTENDEE_BADGE_AVAILABLE:
+            opts.append((self.ATTENDEE_BADGE, 'Full Weekend Pass (${})'.format(self.BADGE_PRICE)))
+        if self.ONE_DAYS_ENABLED and self.PRESELL_ONE_DAYS:
```

Comment:
What this does instead is it looks at the prices you have defined under the [[single_day]] portion of badge prices, then attempts to find badge types with matching names and add them to the dropdown.

So if you have:
```
[[single_day]]
Friday = 45
Saturday = 50
Sunday = 30
```

Then in order to make single-day badges work you need to add e.g. `friday_badge` to the [badge_ranges] section and the [[badge]] section under [enums]. As you can see, this code only executes if you have PRESELL_ONE_DAYS set to True - otherwise it goes back to the old behavior.